interface/patient_file/history/encounters.php

```diff
@@ -41,6 +42,13 @@
     require_once("$srcdir/group.inc");
 }
 
+
+if($GLOBALS['enable_group_therapy'] && !acl_check("groups","glog",false, array('view','write'))){
```

Comment:
Apologies but I am still confused.  $GLOBALS['enable_group_therapy'] appears to be a site-wide global setting. Where is the test for "users with groups acl permission"? Also it seems odd that having a permission will cause you to be denied access to something.